larpandoracontent/LArHelpers/LArClusterHelper.h

```diff
@@ -254,6 +257,16 @@ class LArClusterHelper
     static void GetCaloHitListInBoundingBox(const pandora::Cluster *const pCluster, const pandora::CartesianVector &lowerBound, 
         const pandora::CartesianVector &upperBound, pandora::CaloHitList &caloHitList);
 
+    /**
+     *  @brief  Get the set of the daughter volumes that contains the cluster
+     *
+     *  @param  daughterVolumeIds set of daughter volume Ids pCluster lives in
+     *  @param  pCluster1 address of the cluster
+     *
+     *  @return the set of daughter volumes
```

Comment:
Swap the order of the params here and remove '1' from pCluster. Would just note that daughterVolumeIds is the output variable here and remove the return description